### DISCUSSION TITLE
An inteligent way to view path and filename

### DISCUSSION THREAD

**Discussion by un-lo-so** [sumatrapdfreader/sumatrapdf#3295] [Ideas]
Hello
I search in config file and I don't find anything. 

Suppose to set sumatrapdf to see the full path of open file in tab. Many times the path result too long and will be truncated with "..." and if there are many tab the situation become worst (any tab will be reduce the space left for the path). So, if mouse stay on a tab for 2-3 seconds a tooltip appear with the full path... of course the tolltip could be use the whole screen: so, no problem of left space shouldn't be encountered.

It's possible implement this feature or it exist yet?

**Comment by GitHubRulesOK** [comment]
I find easiest is show short name on tab and then tooltip has long so no problem to know where the file is, If I need to open the path F2 will do that or copy file data use CTRL+D. Its roughly the same in My Browser(s) FileName LongerName&Source or Details (except there D=Save To Favorites!)
![image](https://user-images.githubusercontent.com/12754913/220618273-8a1b7403-00dc-4a95-84de-c3a5ed734156.png)

**Reply by un-lo-so** [reply]
This don't work on my pc... but I have installed sumatrapdf 3.3.something so I uninstal and reinstall the latest version (3.4.6) and it works perfect!

**Comment by igitur** [comment]
We need, not just to view the path/filename, but also copy the full path to clipboard.

**Reply by GitHubRulesOK** [reply]
Always has been Ctrl+D twice but now there is a button to run like "clip"

>CLIP
Description:
    Redirects output of command line tools to the Windows clipboard.
    This text output can then be pasted into other programs.

Or roll your own
![image](https://user-images.githubusercontent.com/12754913/234064514-9f3424e7-e293-49bd-9d59-af6d82bd1d12.png)
```
ExternalViewers [
	[
		CommandLine = C:\windows\system32\cmd.exe /k echo I am "%1" How can I help?
		Name = send &filename to cmd
	]
]
```
